src/main/java/kitchenpos/menu/ui/MenuGroupRestController.java

```diff
@@ -9,24 +9,29 @@
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
+@RequestMapping("/api/menu-groups")
 public class MenuGroupRestController {
+
+    private static final String MAPPING_URL = "/api/menu-groups/";
     private final MenuGroupService menuGroupService;
```

Comment:
```suggestion
    private static final String MAPPING_URL = "/api/menu-groups/";
    
    private final MenuGroupService menuGroupService;
```

클래스 변수와 인스턴스 변수 사이 공백을 추가하면 가독성이 상승합니다!